### ISSUE TITLE
Restrict meaningless `continue` inside loops

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#381]
# Rule request

## Thesis

`continue` can be placed inside a loop like this:

```python
for number in range(10):
     print(number)
     continue
```

But it is meaningless. We should not allow that.
Rule is: if `continue` node is the last one inside loop's body - we should raise a violation.

## Reasoning

This is a best practice. Why would you do that?
